### ISSUE TITLE
heroku pg:backups info fails

### ISSUE THREAD

**Issue by briankung** [heroku/heroku#1458]
When I run `heroku pg:backups info`, the heroku client crashes:

```
505 briankung $ heroku pg:backups info
 !    Heroku client internal error.
 !    Search for help at: https://help.heroku.com
 !    Or report a bug at: https://github.com/heroku/heroku/issues/new

    Error:       undefined method `-' for nil:NilClass (NoMethodError)
    Command:     heroku pg:backups info
    Version:     heroku-toolbelt/3.30.0 (x86_64-darwin10.8.0) ruby/1.9.3
    Error ID:    b42d2bb09a6247ee9df95611f5dfd164


    More information in /Users/briankung/.heroku/error.log
```
The error log says:

```
Heroku client internal error.
undefined method `-' for nil:NilClass
/Users/briankung/.heroku/client/lib/heroku/command/pg_backups.rb:229:in `backup_status'
/Users/briankung/.heroku/client/lib/heroku/command/pg_backups.rb:55:in `backups'
/Users/briankung/.heroku/client/lib/heroku/command.rb:223:in `run'
/Users/briankung/.heroku/client/lib/heroku/cli.rb:40:in `start'
/usr/local/heroku/bin/heroku:24:in `<main>'
```

**Comment by uhoh-itsmaciek** [comment]
Are you still seeing this error? I believe this was fixed in either 3.31.2 or the upcoming 3.31.3 (which you can test by just cloning this repo and invoking `bin/heroku` from the repo root).

**Comment by briankung** [comment]
That works! Thanks.
